README.md

```diff
@@ -108,6 +108,10 @@ Add codacycoverage to your `.travis.yml`:
 ```yml
 # .travis.yml
 language: php
+
+env:
+  - CODACY_PROJECT_TOKEN=<yourTokenGoesHere>
```

Comment:
I've updated the documentation - thanks.
Since I've experienced build timeouts I'm in favour of keeping the retry option as it's recommended by Travis and prevents users from searching for it separately.